definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -12361,6 +12361,20 @@
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "26/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespSolLinkedEmail",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "26/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespSolLinkedEmail",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "R"
+  },
```

Comment:
Service user can read

Reply:
Does beta user already have READ? I believe now caseworkers either have courtadmin or courtadmin_beta role, but not both right?

Reply:
I really only need the service token to read this, not a human/caseworker - is systemupdate better for that?